assets/js/modules/adsense/components/dashboard/DashboardTopEarningPagesWidget.js

```diff
@@ -94,6 +94,37 @@ function DashboardTopEarningPagesWidget( { Widget, WidgetReportZero, WidgetRepor
 		return <WidgetReportZero moduleSlug="analytics" />;
 	}
 
+	const tableColumns = [
+		{
+			title: __( 'Top Earning Pages', 'google-site-kit' ),
+			tooltip: __( 'Top Earning Pages', 'google-site-kit' ),
+			primary: true,
+			Component: ( { row } ) => {
+				const [ title, url ] = row.dimensions;
+				return (
+					<Link
+						href={ url }
+						children={ title }
+						external
+						inherit
+					/>
+				);
+			},
+		},
+		{
+			title: __( 'Revenue', 'google-site-kit' ),
+			tooltip: __( 'Revenue', 'google-site-kit' ),
+			Component: ( { row } ) => numFmt(
+				row.metrics[ 0 ].values[ 0 ],
+				{
+					style: 'decimal',
+					minimumFractionDigits: 2,
+					maximumFractionDigits: 2,
+				}
+			),
+		},
+	];
```

Comment:
Why did you move them into the components? It reads a bit more intuitive, but it also means that the arrays will be regenerated on every render although they're static - or am I missing something here?

Reply:
@felixarntz translations might not be ready/loaded by the time we define that array. By moving that array into the component we make sure that static copy will be translated correctly.

Reply:
@eugene-manuilov Thanks, makes sense!